src/instructlab/model/evaluate.py

```diff
@@ -235,36 +240,41 @@ def launch_server(
     model_name,
     max_workers,
     gpus,
+    backend,
 ) -> tuple:
     # pylint: disable=import-outside-toplevel
     # First Party
-    from instructlab.model.backends import backends
```

Comment:
@cdoern Was there a reason for moving this up top?

Reply:
nm, figured out why:

```
(venv) [ec2-user@ip-10-0-1-95 instructlab]$ time ilab model evaluate --model models/instructlab/granite-7b-lab --judge-model models/instructlab/granite-7b-lab --benchmark mt_bench --max-workers 16 --enable-serving-output
Traceback (most recent call last):
  File "/home/ec2-user/instructlab/venv/bin/ilab", line 5, in <module>
    from instructlab.lab import ilab
  File "/home/ec2-user/instructlab/src/instructlab/lab.py", line 18, in <module>
    from .model import model as model_group
  File "/home/ec2-user/instructlab/src/instructlab/model/model.py", line 11, in <module>
    from .evaluate import evaluate
  File "/home/ec2-user/instructlab/src/instructlab/model/evaluate.py", line 389, in <module>
    type=click.Choice(tuple(backends.SUPPORTED_BACKENDS)),
                            ^^^^^^^^
NameError: name 'backends' is not defined
```

Not sure how big a hit it's having on startup time.

Reply:
I share your concern. :(

Our code is too tightly coupled. My feature proposal #1713 could solve the problem.